docs/src/man/comparisons.md

```diff
@@ -53,61 +59,116 @@ df[findall(==('c'), df.id), :]
-| Transform several columns | `df.agg({'x': max, 'y': min})`                                               | `combine(df, :x => maximum, :y => minimum)`               |
-|                           | `df[['x','y']].mean()`                                                       | `combine(df, [:x, :y] .=> mean)`                          |
-|                           | `df.filter(regex=("^x")).mean()`                                             | `combine(df, r"^x" .=> mean)`                             |
-|                           | `df[['x', 'y']].agg([max, min])`                                             | `combine(df, ([:x, :y] .=> [maximum minimum])...)`        |
-| Multivariate function     | `df.assign(x_y_cor = np.corrcoef(df.x, df.y)[0,1])`                          | `transform(df, [:x, :y] => cor)`                          |
-| Row-wise                  | `df.assign(x_y_min = df.apply(lambda v: min(v.x, v.y), axis=1))`             | `transform(df, [:x, :y] => ByRow(min))`                   |
-|                           | `df.assign(x_y_argmax = df.apply(lambda v: df.columns[v.argmax()], axis=1))` | `transform(df, AsTable([:x,:y]) => ByRow(argmax))`        |
-| DataFrame as input        | `df.groupby('grp').head(2)`                                                  | `combine(d -> first(d, 2), groupby(df, :grp))`            |
-| DataFrame as output       | `df[['x']].agg(lambda x: [min(x), max(x)])`                                  | `combine(:x => x -> (x = [minimum(x), maximum(x)],), df)` |
+This section includes more complex examples.
+
+| Operation                                       | pandas                                                                       | DataFrames.jl                                                                     |
+|-------------------------------------------------|------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|
+| Complex Function                                | `df[['z']].agg(lambda v: np.mean(np.cos(v)))`                                | `combine(df, :z => v -> mean(cos, skipmissing(v)))`                               |
+| Aggregate multiple columns                      | `df.agg({'x': max, 'y': min})`                                               | `combine(df, :x => maximum, :y => minimum)`                                       |
+|                                                 | `df[['x','y']].mean()`                                                       | `combine(df, [:x, :y] .=> mean)`                                                  |
+|                                                 | `df.filter(regex=("^x")).mean()`                                             | `combine(v -> mapcols(mean, v), select(df, r"[xy]"))`                             |
```

Comment:
If https://github.com/JuliaData/DataFrames.jl/pull/2417 adds support for regex this could become:
```suggestion
|                                                 | `df.filter(regex=("^x")).mean()`                                             | `combine(df, names(df, r"^x") .=> mean)`                             |
```

Reply:
OK, I'll take this now assuming that we are putting them on the same release train...